src/core/types/form-validator.ts

```diff
@@ -75,6 +77,7 @@ export type ValidateConfig<T, Err, Dependencies extends any[]> = {
   field: ValidateField<T, Err>;
   triggers?: ValidationTrigger[];
   dependencies?: readonly [...FieldDescTuple<Dependencies>];
+  debounce?: number;
   rules: (
```

Comment:
Lets add missing JSDocs for all properties here - will be quite helpful.
Should explain:
 - difference when passing `Schema.array` vs `Schema.array.nth`
 - effect of specifying triggers (and default - all triggers)
 - effect of specifying dependencies (available in rules cb; extra validation runs when dep changes)
 - effect of specifying debounce (mention that it affects all downstream rules for the field)

Reply:
wdyt of adding it to https://github.com/VirtusLab/formts/issues/52 ?